lib/diagnostic.js

```diff
@@ -9,15 +9,40 @@
 // Write a callback for reduce that selects
 // the minimum value in the array
 const min = function min() {
+    let args;
+    if (arguments[0] instanceof Array) {
+        args = arguments[0];
+    } else {
+        args = arguments;
+    }
+    let answer = args[0];
 
+    for(let i = 1; i < args.length; i++) {
+        if(answer > args[i]) {
+            answer += args[i];
+        }
+        return answer;
+    }
```

Comment:
You misunderstood this question; we were asking you to create a function, `min`, that we could pass into `reduce` as a callback argument.
i.e.
```
[1,2,3].reduce(min);
// => 1
```